SwedbankPaySDK/Classes/SwedbankPaySDK+Extensions/ViewPaymentOrderInfo.swift

```diff
@@ -0,0 +1,118 @@
+
+        /// The `cancelUrl` of the payment order
+        ///
+        /// This url will not be opened in normal operation,
+        /// so it need not point to an actual web page,
+        /// but it must be a valid url and it must be distinct
+        /// form the other urls.
+        var cancelUrl: URL?
+
+        /// The `paymentUrl` of the payment order
+        ///
+        /// The `paymentUrl` you set for your payment order must be a
+        /// Universal Link to your app. When your app receives it in the
+        /// `UIApplicationDelegate.application(_:continue:restorationHandler:)`
+        /// method, you must forward it to the SDK by calling
+        /// `SwedbankPaySDK.continue(userActivity:)`. If it is helpful for your
+        /// systems, you may add extra query parameters to the `paymentUrl`;
+        /// the SDK will ignore these when checking for equality to an ongoing
+        /// payment's `paymentUrl` (the Merchant Backend example does this
+        /// to work around a scenario where Universal Links are not routed
+        /// the way we would wish on iOS 13.3 and below).
+        ///
+        /// Additionally, if your `paymentUrl` is opened in the browser, it must
+        /// ultimately open a url that is otherwise equal to the `paymentUrl`, but
+        /// it has the `callbackScheme` of your `SwedbankPaySDKConfiguration`.
+        /// It may also have additional query parameters, similar to the above.
+        /// When you receive this url in your
+        /// `UIApplicationDelegate(_:open:options:)` method, you must
+        /// forward it to the SDK by calling `SwedbankPaySDK.open(url:)`
+        ///
+        /// Example
+        /// =======
+        ///
+        /// When using the `MerchantBackendConfiguration` and the
+        /// related convenience constructors of `SwedbankPaySDK.PaymentOrderUrls`,
+        /// the actual `paymentUrl`, i.e. this value, will look like this:
+        ///  - https://example.com/sdk-callback/ios-universal-link?scheme=fallback&language=en-US&id=1234
+        ///
+        /// Your `UIApplicationDelegate.application(_:continue:restorationHandler:)`
+        /// can be called either with that url; or if the page was opened in the browser, the
+        /// call will have an extra parameter (this is added by the backend to prevent an infinite loop):
+        ///  - https://example.com/sdk-callback/ios-universal-link?scheme=fallback&language=en-US&id=1234&fallback=true
+        ///
+        /// If neither of the above urls is routed to your app (perhaps because of a broken
+        /// Universal Link configuration), then the page in the brower will instead open
+        /// a url equal to the second one, with the scheme replaced. You will then receive
+        /// an url in your `UIApplicationDelegate(_:open:options:)` method
+        /// that looks like this:
+        ///  - fallback://example.com/sdk-callback/ios-universal-link?scheme=fallback&language=en-US&id=1234&fallback=true
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 122 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)